tests/test_rpc.py

```diff
@@ -457,10 +457,35 @@ def test_list_spend(lianad, bitcoind):
     lianad.rpc.updatespend(res["psbt"])
     lianad.rpc.updatespend(res_b["psbt"])
 
+    # Check 'txids' parameter
+    list_res = lianad.rpc.listspendtxs()["spend_txs"]
+
+    txid = bitcoind.rpc.decodepsbt(list_res[0]['psbt'])['tx']['txid']
+
+    filtered_res = lianad.rpc.listspendtxs(txids=[txid])
+    assert filtered_res['spend_txs'][0]['psbt'] == list_res[0]['psbt']
+    assert len(filtered_res) == 1
+
+    with pytest.raises(RpcError, match="Invalid params: Invalid 'txids' parameter."):
+        lianad.rpc.listspendtxs(txids=[txid, 123])
+
+    with pytest.raises(RpcError, match="Invalid params: Invalid 'txids' parameter."):
+        lianad.rpc.listspendtxs(txids=[0])
+
+    with pytest.raises(RpcError, match="Invalid params: Invalid 'txids' parameter."):
+        lianad.rpc.listspendtxs(txids=[123])
+
+    with pytest.raises(RpcError, match="Invalid params: Invalid 'txids' parameter."):
+        lianad.rpc.listspendtxs(txids=['abc'])
+
+    with pytest.raises(RpcError, match="Invalid params: Invalid 'txids' parameter."):
+        lianad.rpc.listspendtxs(txids=['123'])
+
     # Listing all Spend transactions will list them both. It'll tell us which one has
     # change and which one doesn't.
     list_res = lianad.rpc.listspendtxs()["spend_txs"]
     assert len(list_res) == 2
+
```

Comment:
nit: spurious line break in an unrelated part of the test?

Reply:
fixed